### DISCUSSION TITLE
Website Sidebar - Is it possible to use headings within a .qmd file as sections in the side bar

### DISCUSSION THREAD

**Discussion by Santonia27** [quarto-dev/quarto-cli#7991] [Q&A]
### Description

Im building a website documentation. And instead of creating an individual file for each section, is it possible to create one file with multiple headings/subheadings and create a sidebar navigation using the headings as section ? 

Currently my _quarto.yml works with individual files but I would like to reduce the file number and just work with headings. 

Thanks for your support.

**Comment by mcanouil** [comment]
Sidebar works as described in <https://quarto.org/docs/websites/website-navigation.html#side-navigation>.
There is currently no other way beside you creating a script to fill in the sidebar from a main document, etc.
